server/prisma/migrations/20230718175518_export_type_2/migration.sql

```diff
@@ -5,6 +5,6 @@
 
 */
 -- AlterTable
-ALTER TABLE "Shift" DROP COLUMN "foodTaken",
-ADD COLUMN     "damagedFoodTaken" DOUBLE PRECISION NOT NULL DEFAULT 0,
-ADD COLUMN     "regularFoodTaken" DOUBLE PRECISION NOT NULL DEFAULT 0;
+ALTER TABLE "Shift" DROP COLUMN IF NOT EXISTS "foodTaken",
```

Comment:
uh, wouldn't this be drop column if exists. The other way doesn't make any sense

Reply:
or did prisma make this for you

Reply:
oh yeah i'm dumb